js/src/util.js

```diff
@@ -42,7 +42,7 @@ const Util = (($) => {
         if ($(event.target).is(this)) {
           return event.handleObj.handler.apply(this, arguments) // eslint-disable-line prefer-rest-params
```

Comment:
I don't know but that's related on how to bind special events to a native event with jQuery but you can make a try if you can still listen to `bsTransitionEnd` everything is fine